src/exe/cimbar/cimbar.cpp

```diff
@@ -156,7 +154,7 @@ int decode(const FilenameIterable& infiles, const std::function<int(cv::UMat, un
 				shouldPreprocess = true;
 		}
 
-		int bytes = decodefun(img, color_mode, shouldPreprocess, color_correct);
+		int bytes = decodefun(img, shouldPreprocess, color_correct);
```

Comment:
The general pattern here is basically: if the config defines a param, just rely on the config rather than allowing other trapdoor ways to set it.